.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Thanks, but unfortunately I forgot to mention that you must delete this file too.